### DISCUSSION TITLE
[IntelliSense] Hover Preview for Pug

### DISCUSSION THREAD

**Discussion by Youdaman** [tailwindlabs/tailwindcss#8163] [Ideas]
Hover preview works in Pug (`template lang="pug"` in Vue) but only when using the class attribute, e.g. `div(class="shrink-0")`

Meanwhile the equivalent `div.shrink-0` or `.shrink-0` does not show a preview tooltip on hover.

**Comment by bradlc** [comment]
Hey @Youdaman. I've converted this to a discussion because it's more of a feature request than an issue. I think it would be quite a bit of work to support this but maybe it's something we can look at in the future 👍

**Reply by Youdaman** [reply]
Cheers Brad, it'd definitely be a welcome feature. I'd love to contribute if I understood more about how extensions work, but as a start, the closest thing I can find to part of the code that could be modified to support this would be similar to this commit: https://github.com/tailwindlabs/tailwindcss-intellisense/commit/cfd4e4fd07fc8662cb04db756f83fa18b85a7dc5#diff-6cda51d5e818d971397badc8bc85e57d70108d8bb688905dc81f97940baa9e45 which I think went towards solving https://github.com/tailwindlabs/tailwindcss-intellisense/issues/125

My guess would be changing that regex to look for the string preceded by just a dot might do the trick, or I could be a ways off.
